src/main/plugins/control-panel-plugin/control-panel-plugin.ts

```diff
@@ -26,12 +26,12 @@ export class ControlPanelPlugin implements SearchPlugin {
 
     public execute(searchResultItem: SearchResultItem, privileged: boolean): Promise<void> {
         return new Promise((resolve, reject) => {
-            const shell = new Powershell({});
-            shell.addCommand(`powershell -Command "Show-ControlPanelItem -Name '${searchResultItem.executionArgument}'"`)
-                .then(() => shell.invoke())
-                .then(() => resolve())
-                .catch((reason) => reject(reason))
-                .finally(() => shell.dispose());
+            try {
+                childProcess.execSync(`powershell -Command "Show-ControlPanelItem -Name '${searchResultItem.executionArgument}'"`);
+                resolve();
+            } catch (error) {
+                reject(error);
+            }
```

Comment:
Why using `execSync` instead of `exec`? Won't cause a freeze on the UI?

Reply:
It does not freeze the UI. But it is blocking for the search for the empty string that is performed after pressing enter (because the input text box is cleared). That's why the search spinner is shown until the process execution finishes. It's the same principle as with shell.openItem() (which is blocking as well) in openFile() in file-path-executor.ts.

I know this is hacky and should be replaced by a function that intentially shows the spinner until the execution promise finishes. But because execution of control panel items is quite slow, this is just good for user experience.